packages/frontend/core/src/components/affine/setting-modal/general-setting/editor/edgeless/shape.tsx

```diff
@@ -127,14 +183,212 @@
     [t]
   );
 
+  const docs = useMemo<RadioItem[]>(
```

Comment:
can be moved outside of React component.

Reply:
Added missing i18n for this list.